cypress/e2e/Question/Complete/index.spec.ts

```diff
@@ -0,0 +1,30 @@
+describe('Question Complete screen', () => {
+  context('given the Question Complete screen is opened', () => {
+    beforeEach(() => {
+      cy.signIn();
+      cy.interceptWithDelay('GET', '**/api/v1/surveys/*', 200, {
+        statusCode: 200,
+        fixture: 'Survey/Detail/valid.json',
+      });
+      cy.interceptWithDelay('GET', '**/api/v1/surveys*', 200, {
+        statusCode: 200,
+        fixture: 'Survey/List/valid.json',
+      });
+      cy.interceptWithDelay('GET', '**/api/v1/me', 200, {
+        statusCode: 200,
+        fixture: 'User/valid.json',
+      });
+      cy.interceptWithDelay('POST', '**/responses', 200, {
+        statusCode: 201,
+        fixture: 'Survey/Submit/valid.json',
+      });
+    });
+    it('navigates to the Home screen after 3 seconds', () => {
```

Comment:
Why do you add `after 3 seconds` while the test doesn't mention it? 🤔 
```suggestion
    it('navigates to the Home screen', () => {
```

Reply:
@hoangmirs ah yeah, we don't need to mention it, thanks

Reply:
@hoangmirs removed in [3c85d61](https://github.com/manh-t/react-survey/pull/89/commits/3c85d6154035080dbd9e722241b05ab609e983fc)